From f7e07b7f2b00418368a502d676e7fcefc1eb2fed Mon Sep 17 00:00:00 2001
From: Todd Lipcon <todd@cloudera.com>
Date: Thu, 6 Sep 2012 12:09:40 -0700
Subject: [PATCH 1189/1357] MR1: CLOUDERA-BUILD. embedded jetty may fail to start about 1/5000 times

Upgrades Jetty dependency to 6.1.26.cloudera.2. This addreses
a race condition (JETTY-1316) which caused the Acceptor thread
to sometimes not start, which caused reducer fetch timeouts, etc.

The source for this Jetty release can be found at
https://github.com/cloudera/jetty-hadoop-fix and corresponds to
commit 51042f4a6cd36fe90be7c0f0d208dca5397b527a in that repository.

Author: Todd Lipcon
Ref: CDH-7767, MAPREDUCE-3851
---
 .eclipse.templates/.classpath    |    4 ++--
 ivy/hadoop-core-pom-template.xml |    4 ++--
 ivy/libraries.properties         |    4 ++--
 3 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/.eclipse.templates/.classpath b/.eclipse.templates/.classpath
index fd02b11..12c3be5 100644
--- a/.eclipse.templates/.classpath
+++ b/.eclipse.templates/.classpath
@@ -33,8 +33,8 @@
 	<classpathentry kind="lib" path="build/ivy/lib/Hadoop/common/log4j-1.2.15.jar"/>
 	<classpathentry kind="lib" path="build/ivy/lib/Hadoop/common/mockito-all-1.8.2.jar"/>
 	<classpathentry kind="lib" path="build/ivy/lib/Hadoop/common/oro-2.0.8.jar"/>
-	<classpathentry kind="lib" path="build/ivy/lib/Hadoop/common/jetty-6.1.26.cloudera.1.jar"/>
-	<classpathentry kind="lib" path="build/ivy/lib/Hadoop/common/jetty-util-6.1.26.cloudera.1.jar"/>
+	<classpathentry kind="lib" path="build/ivy/lib/Hadoop/common/jetty-6.1.26.cloudera.2.jar"/>
+	<classpathentry kind="lib" path="build/ivy/lib/Hadoop/common/jetty-util-6.1.26.cloudera.2.jar"/>
 	<classpathentry kind="lib" path="build/ivy/lib/Hadoop/common/jackson-core-asl-1.5.2.jar"/>
 	<classpathentry kind="lib" path="build/ivy/lib/Hadoop/common/jackson-mapper-asl-1.5.2.jar"/>
   	<classpathentry kind="lib" path="build/ivy/lib/Hadoop/common/servlet-api-2.5-6.1.14.jar"/>
diff --git a/ivy/hadoop-core-pom-template.xml b/ivy/hadoop-core-pom-template.xml
index 4079a2d..12ac58b 100644
--- a/ivy/hadoop-core-pom-template.xml
+++ b/ivy/hadoop-core-pom-template.xml
@@ -61,12 +61,12 @@
     <dependency>
       <groupId>org.mortbay.jetty</groupId>
       <artifactId>jetty</artifactId>
-      <version>6.1.26</version>
+      <version>6.1.26.cloudera.2</version>
     </dependency>
     <dependency>
       <groupId>org.mortbay.jetty</groupId>
       <artifactId>jetty-util</artifactId>
-      <version>6.1.26</version>
+      <version>6.1.26.cloudera.2</version>
     </dependency>
     <dependency>
       <groupId>tomcat</groupId>
diff --git a/ivy/libraries.properties b/ivy/libraries.properties
index f9dd3e6..4d5acad 100644
--- a/ivy/libraries.properties
+++ b/ivy/libraries.properties
@@ -54,8 +54,8 @@ jdeb.version=0.8
 jsp.version=2.1
 jsp-api.version=5.5.12
 jets3t.version=0.7.1
-jetty.version=6.1.14
-jetty-util.version=6.1.14
+jetty.version=6.1.26.cloudera.2
+jetty-util.version=6.1.26.cloudera.2
 junit.version=4.8.1
 jdiff.version=1.0.9
 
-- 
1.7.0.4

